app/models/case_request.rb

```diff
@@ -5,10 +5,10 @@ class CaseRequest
     :confirmation_code,
     :fees
 
-  def initialize(opts = {})
-    @case_reference = opts[:case_reference]
-    @confirmation_code = opts[:confirmation_code]
-    @fees = []
+  def initialize(case_reference, confirmation_code)
+    @case_reference    = case_reference
+    @confirmation_code = confirmation_code
+    @fees              = []
```

Comment:
I've got the same thing.  Ultimately, I've found it just makes more work.  However, if that's the consensus, then I won't object.